ci/integration_tests/README.md

```diff
@@ -0,0 +1,44 @@
+# Dao Dao Integration Tests
+
+Dao Dao e2e integration tests with gas profiling.
+
+## Running Locally
+
+TODO: Add a just file to make this all easy
+
+### Hitting Local Juno
+* `./scripts/deploy_local.sh juno10j9gpw9t4jsz47qgnkvl5n3zlm2fz72k67rxsg`
+* `tail -n 1 default-account.txt | junod keys add localval --recover`
+* `cd ci/integration_tests`
+* `CONTRACT_DIR="../../artifacts" GAS_REPORT_OUT="gas_report.json" CONFIG="configs/local.yaml" RUST_LOG=debug cargo test`
+
+### Hitting Testnet
+* `cd ci/integration_tests`
+* Configure `config.yaml` with your junod testnet key name
+* `CONTRACT_DIR="../../artifacts" GAS_REPORT_OUT="gas_report.json" CONFIG="configs/testnet.yaml" RUST_LOG=debug cargo test`
+
+
+## Adding New Integration Tests
+
+```rust
+#[test]
+fn new_dao_has_no_items() {
+     let dao = create_dao(
+        Some(admin_addr.clone()),
+        admin_addr,
+        "cw20_staked_balance_voting",
+        "cw_proposal_single",
```

Comment:
Do these correspond to names of `.wasm` files? Might be nice to include that info here. :)

Reply:
Agreed, I need to at least add a comment like I did below with:
```
// NOTE: `cw_core` wasm was stored in test_harness::test_runner.rs:setup()
```

but I need to make this more straightforward to use in general now that the cosm-orc v1 usage has solidified here.

Reply:
Im actually just removing these params, because as `create_dao()` is currently written it really needs to just be these strings so Im going to just use them in the test. Later on I need to make `create_dao()` take in optional overrides for each of the fields once we need tests with different modules and configurations.